### DISCUSSION TITLE
How close to the SSL certificate expiration time does Vercel renew it?

### DISCUSSION THREAD

**Discussion by wereHamster** [vercel/vercel#9718] [General] [answered]
### Description

… for certificates issued by ACME, that is. We currently have a monitor that warns us when the expiration time is 4 weeks in the future. I assume Vercel decides to renew the certificate closer to the expiration time (3 weeks, 2 weeks?). 

From the Let's Encrypt FAQ:

> What is the lifetime for Let’s Encrypt certificates? For how long are they valid?
>
> Our certificates are valid for 90 days. You can read about why [here](https://letsencrypt.org/2015/11/09/why-90-days.html).  There is no way to adjust this, there are no exceptions. We recommend automatically renewing your certificates every 60 days.

**Comment by wiiiimm** [comment, marked as answer]
You can find this on [Vercel's docs](https://vercel.com/guides/renewal-of-ssl-certificates-with-a-vercel-domain):

> Vercel attempts to renew this certificate automatically within a 30 day period before it expires. While most certificates are renewed at the 30 day mark, it is not unusual for certificates to be renewed at a later date due to a combination of factors. If your domain is misconfigured and the certificate cannot be renewed as a result, Vercel will contact you via email to notify you of this.

SSLs are renewed automatically 0 days to 30 days before expiry.
